ProjectCoimbra.UWP/Project.Coimbra/Pages/DurationPage.xaml.cs

```diff
@@ -19,7 +19,7 @@ public sealed partial class DurationPage : Page
             this.InitializeComponent();
 
         private void BackButton_Click(object sender, RoutedEventArgs e) =>
-            _ = this.Frame.Navigate(UserData.GameMode == UserData.Mode.Solo ? typeof(SongPage) : typeof(BandSongPage), null, new DrillInNavigationTransitionInfo());
+            _ = this.Frame.Navigate(UserData.GameMode == UserData.Mode.Online ? typeof(BandSongPage) : typeof(SongPage), null, new DrillInNavigationTransitionInfo());
```

Comment:
Are we sure that this works for all modes? I find it a little strange that just changing this resolves the error and doesn't break anything else, but I haven't tested it (or used it in a while) so I can't be sure.

Reply:
Yes, I am pretty sure. This is a routing error.
There is no route involving BandSongPage for Offline mode.